ui/app/src/components/ApprovedScheduledDashlet/ApprovedScheduledDashlet.tsx

```diff
@@ -168,17 +168,14 @@ export function ApprovedScheduledDashlet() {
 
   // region Item Updates Propagation
   useEffect(() => {
-    const events = [itemsDeleted.type, itemsRejected.type, itemsApproved.type, itemsScheduled.type];
+    const events = [deleteContentEvent.type, workflowEvent.type, publishEvent.type];
     const hostToHost$ = getHostToHostBus();
     const subscription = hostToHost$.pipe(filter((e) => events.includes(e.type))).subscribe(({ type, payload }) => {
       switch (type) {
-        case itemsApproved.type:
-        case itemsScheduled.type:
-        case itemsDeleted.type:
-        case itemsRejected.type: {
-          if (payload.targets.some((path) => state.itemsLookup[path])) {
-            refresh();
-          }
+        case deleteContentEvent.type:
+        case workflowEvent.type:
+        case publishEvent.type: {
+          refresh();
```

Comment:
The whole switch is kind of pointless as the handling is same for all listened events. Check other places changed for this same situation. Might be easier to locate these here on the PR files tab.